nixos/modules/services/web-apps/discourse.nix

```diff
@@ -824,8 +827,14 @@ in
       upstreams.discourse.servers."unix:/run/discourse/sockets/unicorn.sock" = {};
 
       appendHttpConfig = ''
+        # Additional MIME types that nginx should handle
```

Comment:
Our nginx uses mailcap mime.types which already includes those.

Reply:
Ah, indeed we do. Removed. :)